styleguide.config.js

```diff
@@ -25,82 +26,69 @@ module.exports = {
-        }, {
-          name: 'Form Page Type',
-          content: path.join(__dirname, 'pages/design-patterns/pages/form-type.md'),
-        }, {
-          name: 'Form/Table Hybrid Page Type',
-          content: path.join(__dirname, 'pages/design-patterns/pages/hybrid-type.md'),
-        }, {
-          name: 'Project View Page Type',
-          content: path.join(__dirname, 'pages/design-patterns/pages/project-view-type.md'),
-        }],
-      }],
-    },
-  ],
-  skipComponentsWithoutExample: true,
+  sections: [{
+    name: 'README',
+    content: path.join(__dirname, './README.md'),
+  }, {
+    name: 'Living Codebase',
+    href: 'https://github.com/mavenlink/design-system',
+  }, {
+    name: 'Living Style Guide',
+    href: 'https://www.notion.so/Mavenlink-Design-System-d3118c3e2f5647d6adfc5b1599af993e',
+  }, {
+    name: 'Form Controls',
+    components: [
+      path.join(__dirname, 'src/components/form/form.jsx'),
+      path.join(__dirname, 'src/components/form-control/form-control.jsx'), // Is this private? Should it be undocumented as implementation details?
+      path.join(__dirname, 'src/components/form-control-icons/form-control-icons.jsx'), // Is this private? Should it be undocumented as implementation details?
+      path.join(__dirname, 'src/components/autocompleter/autocompleter.jsx'),
+      path.join(__dirname, 'src/components/checkbox/checkbox.jsx'),
+      path.join(__dirname, 'src/components/custom-field-input-currency/custom-field-input-currency.jsx'),
+      path.join(__dirname, 'src/components/custom-field-input-date/custom-field-input-date.jsx'),
+      path.join(__dirname, 'src/components/custom-field-input-multiple-choice/custom-field-input-multiple-choice.jsx'),
+      path.join(__dirname, 'src/components/custom-field-input-number/custom-field-input-number.jsx'),
+      path.join(__dirname, 'src/components/custom-field-input-single-choice/custom-field-input-single-choice.jsx'),
+      path.join(__dirname, 'src/components/custom-field-input-text/custom-field-input-text.jsx'),
+      path.join(__dirname, 'src/components/date/date.jsx'),
+      path.join(__dirname, 'src/components/input/input.jsx'),
+      path.join(__dirname, 'src/components/multi-autocompleter/multi-autocompleter.jsx'),
+      path.join(__dirname, 'src/components/multi-select/multi-select.jsx'),
+      path.join(__dirname, 'src/components/number/number.jsx'),
+      path.join(__dirname, 'src/components/percentage/percentage.jsx'),
+      path.join(__dirname, 'src/components/select/select.jsx'),
+      path.join(__dirname, 'src/components/textarea/textarea.jsx'),
+    ],
+  }, {
+    name: 'Widgets',
+    components: [
+      path.join(__dirname, 'src/components/button/button.jsx'),
+      path.join(__dirname, 'src/components/calendar/calendar.jsx'),
+      path.join(__dirname, 'src/components/help-icon/help-icon.jsx'),
+      path.join(__dirname, 'src/components/icon-button/icon-button.jsx'), // Should this be part of a Button interface?
```

Comment:
Good question. I think even if we go that route, we'd probably still want a separate component since the DOM structure is so different. Maybe `Button` could just implement a prop that switches to this implementation in that case?